src/Problem1Class.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+class Problem1Class
+{
+    protected string $test;
+
+    public function test(): string
+    {
+        $functionsCookies = $_COOKIE;
```

Comment:
:no_entry_sign: **Error**: Due to server-side caching, server-side based client related logic might not work. We recommend implementing client side logic in JavaScript instead (*WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___COOKIE*).